README.md

```diff
@@ -259,6 +259,36 @@ declare module 'fastify' {
 }
 ```
 
+## Provider Quirks
+
+This is a list of known quirks of some providers that needed additional work to get everything set up correctly.
+
+### Twitch
+
+Twitch requires that the request for a token in the oauth2 flow contains the `client_id` and `client_secret` property.
+Unfortunately, this library cannot set this up automatically. Howevever, it's easy to configure:
```

Comment:
Suggested change. (Sometimes it's good to point out what you're doing in both the text and the code example, as I know some devs read one or the other.)

"When authenticating via Twitch, include the `client_id` and `client_secret` in `tokenRequestParams`:"

Reply:
thanks, applied!